extensions/walletProviders/utxo/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@tatumio/utxo-wallet-provider",
```

Comment:
idk if it was discussed, but what if we'll introduce naming like: `@tatumio/extension/utxo-wallet-provider` or `@tatumio/wallet-provider/utxo`. or sdk-extension, or sdk-wallet-provider to have some obvious link to sdk.
It is global question, not so much related to these changes :)

Reply:
We already have tron-wallet-provider and evm-wallet-provider. Maybe README is enough? Need to discuss with PM and devrel